superset-frontend/src/components/ListView/ListView.tsx

```diff
@@ -48,8 +48,24 @@ const ListViewStyles = styled.div`
     padding-bottom: 48px;
 
     .body {
-      overflow: scroll;
+      overflow: auto;
       max-height: 64vh;
+
+      &::-webkit-scrollbar {
+        -webkit-appearance: none;
+        &:vertical {
+          width: 7px;
+        }
+        &:horizontal {
+          height: 7px;
+        }
+      }
+
+      &::-webkit-scrollbar-thumb {
+        border-radius: 4px;
+        background-color: rgba(0, 0, 0, 0.5);
+        -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
```

Comment:
Aha! It's in Emotion now! I made some comments on https://github.com/apache/incubator-superset/pull/10982 about these color/pixel values, and how they might be updated to leverage the Theme. If those values are used here, it seems that other PR might not need all the LESS code at all?

Reply:
Happy to copy over the theme bits as "suggestions" here so you can directly commit them, if it helps.

Reply:
Done